nengo_bones/config.py

```diff
@@ -5,6 +5,30 @@
 
 import yaml
 
+NOTICE = """# NOTICE:  All information contained herein is and remains the property
+# of Applied Brain Research. The intellectual and technical concepts contained herein
+# are proprietary to Applied Brain Research and may be covered by U.S. and Foreign
+# Patents, patents in process, and are protected by trade secret or
+# copyright law. Dissemination of this information or reproduction of
+# this material is strictly forbidden unless prior written permission is
+# obtained from Applied Brain Research. Access to the source code contained herein is
+# hereby forbidden to anyone except current Applied Brain Research employees,
+# contractors or other outside parties that have executed Confidentiality
+# and/or Non-disclosure agreements explicitly covering such access.
+#
+# The copyright notice above does not evidence any actual or intended
+# publication or disclosure  of  this source code, which includes
```

Comment:
There's extra spaces in between words here, which are likely from some original source that tried to have everything justified even though we're in plaintext. I would remove those extra spaces and use some tool to wordwrap it to 80 characters (we usually allow more, but 80 to be safe is probably best).